sample/README.md

```diff
@@ -1,6 +1,61 @@
-<h1>Hazelcast Go Client</h1>
+# Hazelcast Go Client Samples
 
-<h2> Examples</h2>
+In order to run any example, you should have a running Hazelcast instance. If you don't already have one:
+- [download the latest hazelcast release](https://hazelcast.org/download/), 
+- [start an instance](http://docs.hazelcast.org/docs/latest/manual/html-single/index.html#using-the-scripts-in-the-package).
 
-- <h3>hello-world</h3>
-        "<i>Hello World</i>", the classic entry point to learning
+When you have a running Hazelcast instance, you will see its address and port in the console.  You should add that information to the **network config**, as shown below,
+
+```
+config := hazelcast.NewConfig()
+config.NetworkConfig().AddAddress("192.168.1.9:5701") // change this with your address
+
+```
+
+**The address given above is an example. Don't forget to change it to your server's address.**
+## Samples
+
+- ###[flakeidgen](https://github.com/hazelcast/hazelcast-go-client/tree/master/sample/flakeidgen)
```

Comment:
the hashtags are not needed at here(and below)

Reply:
@asimarslan  Why are they not needed?